### PR TITLE
Fix issue where plugin install always install amd64 for plugins.

### PR DESCRIPTION
**Pull request by binaek**

### CODE REVIEW

File: ociinstaller/steampipeimage.go [hunk 1]
```diff
@@ -186,12 +186,13 @@ func getPluginImageData(layers []ocispec.Descriptor) (*PluginImage, error) {
...
-			log.Println("[TRACE] falling back to the next one, if any")
-			continue
+		if len(foundLayers) == 1 {
+			res.BinaryFile = foundLayers[0].Annotations["org.opencontainers.image.title"]
+			break
 		}
-		res.BinaryFile = foundLayers[0].Annotations["org.opencontainers.image.title"]
+		log.Println("[TRACE] could not find data for", mediaType)
+		log.Println("[TRACE] falling back to the next one, if any")
+		continue
```
Comment (kaidaguerre) [inline ociinstaller/steampipeimage.go:195]: continue is not needed

File: ociinstaller/steampipeimage.go [hunk 2]
```diff
@@ -186,12 +186,13 @@ func getPluginImageData(layers []ocispec.Descriptor) (*PluginImage, error) {
 	// get the binary plugin file info
 	for _, mediaType := range MediaTypeForPlatform("plugin") {
```
Comment (kaidaguerre) [inline ociinstaller/steampipeimage.go:187]: this code is a bit opaque - at least out of context like this

Add a short comment to explain thew control flow

**Review by kaidaguerre — CHANGES_REQUESTED** [review]

File: ociinstaller/steampipeimage.go (hunk 2 above)
Comment (binaek) [reply ociinstaller/steampipeimage.go:188]: done

File: ociinstaller/steampipeimage.go (hunk 1 above)
Comment (binaek) [reply ociinstaller/steampipeimage.go]: :-D

**Pull request merged**